qiskit/synthesis/clifford/clifford_decompose_greedy.py

```diff
@@ -36,7 +36,7 @@ def synth_clifford_greedy(clifford):
     greedy Clifford compiler that is described in Appendix A of
     Bravyi, Hu, Maslov and Shaydulin.
 
-    This method typically yields better CX cost compared to the Aaronson-Gottesma method.
+    This method typically yields better CX cost compared to the Aaronson-Gottesman method.
```

Comment:
Other reviewers, search results suggest it indeed should be Gottesman but FYI this one is not as obvious.

Reply:
I lucked out that I have heard this name bandied about in quantum contexts so I had a pretty high degree of confidence  on it 🤣thanks for looking so closely @Eric-Arellano !